src/stream/src/executor/lookup/impl_.rs

```diff
@@ -166,10 +165,10 @@ impl<S: StateStore> LookupExecutor<S> {
         }
 
         // compute the arrange keys used for the lookup
-        let arrangement_order_types = arrangement_order_rules
-            .iter()
-            .map(|x| x.order_type)
-            .collect();
```

Comment:
This arrangement_order_types do not includes pk but only sort key. This is different from Materialize executor. I investigate and found that it's because we use arrangement key here: 

https://github.com/singularity-data/risingwave/blob/e26eb0c0218f499a25c7c35e4ee0315c24eca2f1/src/stream/src/from_proto/lookup.rs#L35-L40

It should be changed to 

https://github.com/singularity-data/risingwave/blob/e26eb0c0218f499a25c7c35e4ee0315c24eca2f1/src/stream/src/from_proto/mview.rs#L78-L83

Changed in this PR or later are both OK